db/migrate/20221122095639_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Please rename the `user` foreign key column to `author` instead, as per the following [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).